plugins/woocommerce/client/legacy/css/admin.scss

```
@@ -5565,45 +5582,528 @@ img.help_tip {
+	#poststuff #titlediv .inside {
+		padding: 0;
+		margin-top: $gap-smaller;
+	}
+	#edit-slug-box {
+		padding: 0;
+		margin: 0;
+	}
+	#titlediv #title {
+		margin: 0;
+	}
+
+	// Postbox cards.
+	.postbox {
+		border: 0;
+		border-radius: $border-radius;
+		display: flex;
+		flex-direction: column;
+	}
+	// Mimic wider gap between postboxes.
+	.post-body-content,
+	.postbox {
+		margin-bottom: 28px;
+	}
+	#postdivrich {
+		margin-top: 28px;
+		.wp-editor-wrap {
+			margin: $gap $gap 0;
+		}
+		#post-status-info {
+			margin: 0px $gap $gap;
+			width: calc(100% - 2 * $gap);
+		}
+	}
+	//
+
+	// Post/meta boxes.
+	.postbox-header {
+		padding: 12px 12px 12px 16px;
+		flex-shrink: 0;
+		align-self: stretch;
+		border-color: $separator-color;
+		min-height: 30px;
+
+		.hndle {
+			gap: $gap-small;
+			flex-wrap: wrap;
+		}
+	}
+	.closed .postbox-header {
+		border: 0;
+	}
+
+	.inside,
+	#commentsdiv.postbox .inside,
+	#postimagediv .inside {
+		padding: 16px;
+
+		// Remove margins of paragraph wrappers that probably should have been divs separated with gap.
+		p:first-child {
+			margin-top: 0;
+		}
+		p:last-child {
+			margin-bottom: 0;
+		}
+	}
+
+	// Separate rule, not to overspecify the above ones with id selector.
+	#poststuff {
+		h2.postbox-header{
+			padding: 12px 16px;
+		}
+		.hndle {
+			padding: 0;
+		}
+
+		.inside {
+			margin: 0;
+		}
+	}
+
+	.wp-editor-tools {
+		display: flex;
+		flex-wrap: wrap;
+		justify-content: space-between;
+	}
+
+	// Rich editor tabs.
+	.wp-editor-tabs {
+		display: flex;
+		align-self: flex-end;
+
+		.wp-switch-editor {
+			float: none;
+			background: none;
+			margin: 0;
+			padding: 10px $gap;
+			color: var(--Alias-text-text-tertiary, $alias-text-tertiary);
+			border-width: 0 0 2px 0;
+			border-bottom-color: transparent;
+			justify-content: center;
+			align-items: center;
+		}
+	}
+	.html-active .switch-html,
+	.tmce-active .switch-tmce {
+		color: var(--Alias-text-text, $alias-text);
+	}
+
+
+	.wp-media-buttons {
+		display: flex;
+		align-self: flex-start;
+		flex-wrap: wrap;
+		gap: $gap-small;
+		padding-bottom: $gap;
+	}
+	.wp-editor-tools:after {
+		content: none;
+		display: none;
+	}
+
+	#postexcerpt >.postbox-header > .hndle >.woocommerce-help-tip,
+	#wp-content-media-buttons > .woocommerce-help-tip {
+		margin: 0;
+	}
+
+	// Product/coupon data options panel.
+	.woocommerce_options_panel {
+		.options_group:not(.hidden),
+		// General coupos data does not have options group wrapper, make it look like it has.
+		&#general_coupon_data:not(.hidden) {
+			display: flex;
+		}
+		.options_group,
+		&#general_coupon_data {
+			border-bottom: 1px solid #EDEDED;
+			padding: 36px 48px;
+			display: flex;
+			flex-direction: column;
+			gap: $gap-small;
+		}
+		/*
+		 * Sold individually option has help tip icon as a direct child, not as a child of form-field paragraph.
+		 * Let's make it look as it would be structured as others.
+		 */
+		 .options_group.inventory_sold_individually {
```

There is an extra space in the indentation.

```suggestion
		.options_group.inventory_sold_individually {
```